juriscraper/opinions/united_states/federal_special/uscgcca.py

```diff
@@ -0,0 +1,76 @@
+# Scraper for US Coast Guard Court of Criminal Appeals
+
+from juriscraper.lib.string_utils import convert_date_string
+from juriscraper.OpinionSite import OpinionSite
+from re import search
+from requests.utils import requote_uri
+
+
+class Site(OpinionSite):
+    def __init__(self, *args, **kwargs):
+        super(Site, self).__init__(*args, **kwargs)
+        self.court_id = self.__module__
+        self.mj_cite_regex = "\\d+ *M\\.*J\\.* *\\d+"
+        self.wl_cite_regex = "\\d+ *W\\.*L\\.* *\\d+"
+        self.current_page = 1
+        self.base_url = (
+            "https://www.uscg.mil/Resources/Legal/"
+            "Court-of-Criminal-Appeals/"
+            "CGCCA-Opinions/smdpage15701/{}"
+        )
+        self._format_url()
+
+    def _format_url(self):
+        self.url = self.base_url.format(self.current_page)
+
+    def _process_html(self):
+        table_xpath = "//table[@class='Dashboard']//tbody[1]"
+        self.html = self.html.xpath(table_xpath)[0]
+
+    def _get_case_dates(self):
+        dates_xpath = "//tr//td[3]"
+        return [
+            convert_date_string(date.text_content())
+            for date in self.html.xpath(dates_xpath)
+        ]
+
+    def _get_case_names(self):
+        names_xpath = "//tr//td[1]"
+        names_regex = "^([a-zA-Z ]+ [Vv]\\.? [ ]*[a-zA-Z]+)"
+        names = []
+        for name in self.html.xpath(names_xpath):
+            text = name.text_content()
+            m = search(names_regex, text)
+            names.append(m.group(0) if m else text)
+
+        return names
+
+    def _get_precedential_statuses(self):
+        publish_regex = "((- [a-zA-Z]+)|(\\([a-zA-Z ]+\\)))$"
+        statuses = []
+        for name in self._get_case_names():
+            m = search(publish_regex, name)
+            statuses.append(m.group(0).strip("()- ") if m else "PUBLISHED")
```

Comment:
PUBLISHED should be Published.

Reply:
Actually, this requires a deeper dive.  And it's one that I didn't recognize (mostly) until now.  

<img width="432" alt="Screen Shot 2021-04-13 at 2 36 30 PM" src="https://user-images.githubusercontent.com/6464529/114604376-cde70a00-9c66-11eb-85b0-095d9fa9f589.png">

I'm attaching a screenshot of the documents currently available for help in dissecting this though.

When we say something is published, we don't mean is it available or published online.  We ask is it precedential (having the authority to bind lower courts).  Not all opinions carry the same weight or are binding.  

The easiest way to tell if someone is precedential here is if it gets "reported" in an official reporter.  In this case, I believe the official reporter is the Military Journal (MJ).  You can have a west, lexis, neutral and MJ citation but the MJ citation is basically what denotes that it is Published and precedential.  In other scrapers, you could assign pretty much all full opinions as published if its a Supreme Court for example.  But even there we wouldn't classify orders and minor documents precedential.  

So for the Coast Guard, I think you can check if the document has an MJ citation and also that it is not an opinion.  If those two parts are satisfied, set it as precedential (`Published`).  If it has an MJ citation but is an order, I would hedge and mark it as `Unknown`, and otherwise I would say non-precedential (`Unpublished`)